src/ui/public/doc_table/components/table_row.js

```diff
@@ -128,7 +147,7 @@ module.directive('kbnTableRow', function ($compile) {
           // rebuild cells since we modified the children
           $cells = $el.children();
 
-          if (i === 0 && !reuse) {
+          if (!reuse) {
             $toggleScope = $scope.$new();
             $compile($target)($toggleScope);
```

Comment:
Sorry for the late and random comment, but doesn't this `$compile($target)($toggleScope);` on all "newHtmls" elements for each row introduce some performance degradation when showing many rows ?